app/helpers/application_helper.rb

```diff
@@ -276,4 +276,148 @@ def sign_in_path
   def unread_notifications(user)
     Comment.accessible_by(current_ability).find_since_last_login(user)
   end
+
+  # Recieves a PaperTrail::Version object
+  # Outputs the list of attributes that were changed in the version (ignoring changes from one blank value to another)
+  # Eg: If version.changeset = '{"title"=>[nil, "Premium"], "description"=>[nil, "Premium = Super cool"], "conference_id"=>[nil, 3]}'
+  # Output will be 'title, description and conference'
+  def updated_attributes(version)
+    version.changeset.
+      reject{ |_, values| values[0].blank? && values[1].blank? }.
+      keys.map{ |key| key.gsub('_id', '').tr('_', ' ')}.join(', ').
+      reverse.sub(',', ' dna ').reverse
+  end
+
+  def name_or_username(user_id)
```

Comment:
It doesn't check if its blank.
[https://github.com/openSUSE/osem/blob/master/app/models/user.rb#L75](https://github.com/openSUSE/osem/blob/master/app/models/user.rb#L75) only checks if user.name is nil